x-pack/plugin/eql/build.gradle

```diff
@@ -8,11 +8,6 @@ esplugin {
   extendedPlugins = ['x-pack-ql', 'lang-painless']
 }
 
-ext {
-  // EQL dependency versions
-  antlrVersion = "4.9.2"
-}
-
```

Comment:
I was thinking that now that painless is up-to-date, there is no reason to declare a specific version for the query languages plugins and we can rather use the main declared version (recently added).